packages/app/cypress/e2e/top-nav.cy.ts

```diff
@@ -417,7 +417,7 @@ describe('App Top Nav Workflows', () => {
             return new Promise((resolve) => {
               setTimeout(() => {
                 resolve(options.user)
-              }, 1000)
+              }, 2000)
```

Comment:
Is there some sort of more intelligent waiting we can do, rather than "two seconds"?

Reply:
Umm, nothing great that I can think of. This is a `cy.task` so it's run in node which makes it a little more difficult. I potentially could create another task that informs the first one to finish logging in after we've verified the "Waiting for you to log in" text but not sure that's worth it. I could also just remove the check for the "Waiting for you to log in" text and rely solely on the component test.